assets/css/_logged_in_as.scss

```diff
@@ -0,0 +1,27 @@
+@use "./color/tokens_2024" as tokens;
+
+.c-logged-in-as__container {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
```

Comment:
**nitpick:** this is probably less of an issue two years on, but we have run into problems with `gap` in flexbox in the past. See #1541.

Reply:
We really should get caniuse in CI (again?)

Reply:
Oof! Browser incompatibility strikes again!

Would you recommend switching to something like `margin-right: 0.5rem` on the icon?

Reply:
Or (I think equivalently) `margin-left: 0.5rem` on the text?

Reply:
https://github.com/mbta/skate/pull/2383/commits/04458340325ba72ec0daefe42ae7a89574684d13